src/hooks/useOrgHooks.js

```diff
@@ -63,19 +61,10 @@ export function useOrganzation() {
   useEffect(() => {
     let cancelled = false
     const fetchOrg = async () => {
-      const organization = await connect(
-        ORG_ADDRESS,
-        [
-          'thegraph',
-          {
-            orgSubgraphUrl: ORG_SUBRAPH_URL,
-          },
-        ],
-        {
-          readProvider: ethereum || ethers,
-          chainId: getDefaultChain(),
-        }
-      )
+      const organization = await connect(ORG_ADDRESS, 'thegraph', {
+        ethereum,
+        network: getDefaultChain(),
```

Comment:
`chainId` is now `network`. It can also now accept [a chain ID, a network name, or a network object](https://github.com/aragon/connect/blob/bf0eec72b9320d9166e1cd5717dc1f9f4a0b8899/packages/connect-types/src/index.ts#L9-L12).